testing/influxdb/APKBUILD

```diff
@@ -1,74 +1,60 @@
 # Contributor: Stefan Wagner <stw@bit-strickerei.de>
 # Maintainer: Stefan Wagner <stw@bit-strickerei.de>
 pkgname=influxdb
-pkgver=1.3.6
+pkgver=1.4.2
 pkgrel=0
 pkgdesc="Scalable datastore for metrics, events, and real-time analytics"
 url="https://www.influxdata.com/time-series-platform/influxdb/"
 arch="all !aarch64"
 license="MIT"
-makedepends="go glide python2 asciidoc xmlto"
+makedepends="go python2 asciidoc xmlto"
 pkgusers="influxdb"
 pkggroups="influxdb"
 install="$pkgname.pre-install"
+options="!check"
 subpackages="$pkgname-doc"
-source="influxdb.initd
+source="$pkgname-$pkgver.tar.gz::https://github.com/influxdata/$pkgname/archive/v$pkgver.tar.gz
+	influxdb.initd
 	influxdb.logrotate
-	glide.yaml
-	glide.lock
 	"
 builddir="$srcdir/src/github.com/influxdata/$pkgname"
-_git_repo="https://github.com/influxdata/influxdb.git"
 
-unpack() {
-	export GOPATH="$srcdir"
-	git clone --branch v${pkgver} $_git_repo $builddir || return 1
-	cd "$builddir"
-	cp "$srcdir"/glide.yaml "$srcdir"/glide.lock . || return 1
-	glide install --skip-test || return 1
-	default_unpack || return 1
+prepare() {
+	mkdir -p ${builddir%/*}
+	mv "$srcdir"/$pkgname-$pkgver "$builddir"/ || return 1
+	default_prepare
 }
 
 build() {
 	export GOPATH="$srcdir"
 	cd "$builddir"
-	python2 build.py --no-get || return 1
+	go get github.com/sparrc/gdm
```

Comment:
Using `go get` is generally a bad idea since it simply fetches the HEAD of the git repository and therefore makes the build irreproducable unless they have a `vendor/` directory in `$builddir`. Please consider packing `gdm` instead (if it isn't already).